### PR TITLE
remove PEP498  f-string new feature for support  python3.5

### PR DESCRIPTION
**Pull request by KimBioInfoStudio**
As Forum https://discuss.tvm.ai/t/what-is-minimal-python3-version-requirements/4594
Just remove  pep498  which is a new feature in python 3.6+ 
DONE:
- replace f-strings with format
TODO:
- add codestyle check for python3.5

could u kindly have a look at this pr? @tqchen  @yongwww @MarisaKirisame

### CODE REVIEW

**Comment by tqchen** [comment]
please fix the CI error

**Comment by KimBioInfoStudio** [comment]
> please fix the CI error

```make: *** [pylint] Error 16

Makefile:77: recipe for target 'pylint' failed

script returned exit code 2
```
I think this ci error not cause by my code?

File: python/tvm/relay/_parser.py [hunk 1]
```diff
@@ -252,14 +252,14 @@ def _check_existing_typ_expr(self, name, new_expr):
             new_typ_name = self._type_expr_name(new_expr)
             existing_typ_name = self._type_expr_name(self.global_type_vars[name])
             raise ParseError(
-                f"{new_typ_name} `{name}` conflicts with existing {existing_typ_name}")
+                f"{0} `{1}` conflicts with existing {2}".format({new_typ_name, name, existing_typ_name}))
```
Comment (zhuochenKIDD) [inline python/tvm/relay/_parser.py]: From CI Error: 

python/tvm/relay/_parser.py:255: [C0301(line-too-long), ] Line too long (105/100)

@KimBioInfoStudio perhaps you need to make pylint happy?

Comment (KimBioInfoStudio) [reply python/tvm/relay/_parser.py]: fine, have fix

**Comment by tqchen** [comment]
Thanks @KimBioInfoStudio

**Pull request merged**